CHANGELOG.md

```diff
@@ -3,6 +3,87 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.15.0
+
+_Sep 22, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add support for `getRowsToExport` option to print export on the data grid (#10084) @zreecespieces
+- 🌍 Improve Finnish (fi-FI) locale on the data grid
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### Data Grid
+
+#### `@mui/x-data-grid@6.15.0`
+
+- [DataGrid] Add support for `getRowsToExport` option to print export (#10084) @zreecespieces
+- [DataGrid] Fix dev warning about `InputLabelProps` (#10413) @romgrk
+- [DataGrid] Refactor `GridMenu` prop `onClickAway` to `onClose` (#10411) @romgrk
+- [DataGrid] Restore focus after `GridMenu` closes (#10412) @romgrk
+- [DataGrid] Fix typing of `GridActionsCellItem` (#10344) @romgrk
+- [DataGrid] Hide `eval` from bundlers (#10329) @romgrk
+- [DataGrid] Add `border: 0` to unmounted focused cell to avoid layout shifts in that row (#10318) @lauri865
+- [l10n] Improve Finnish (fi-FI) locale (#10346) @samijouppila
```

Comment:
```suggestion
```

Reply:
This is getting old. 🙈 
WDYT about one of these options:
1. Do a double prefix (i.e., `[l10n][pickers] Improve <...>`) so that the script could easily understand which package this PR belongs to
2. Keep only the package prefix as we always include the `locale` word in the PR title anyway. 🤷 🙈 
3. Keep the process as is, but add extra step to the `releaseChangelog` so that it would fetch each `l10n` commit related PR information and check it's labels to understand the relevant package.

Reply:
1. makes sense to me. Though, it could be `[pickers] Improve Finnish (fi-FI) locale`.

Reply:
So, it seems that we are gravitating between 1st and 2nd options.
If we are going with the 1st options then there are a few follow-up questions:
1. What is the tag order? `[l10n][pickers]` or the other way around? MUI Core seems to use `[component][product]` order, based on that, it would possibly make more sense to do `[pickers][l10n]`...
2. Would we keep the `[pickers]` or `[DataGrid]` or any other component part in the final release changelog? Or would it only serve as a marker for us when formatting the log itself? 🤔 I'll remind that we already split changes per package, thus also including the `[product]` tag would sort of be a duplciate 🤷  

cc @alexfauquette as you've voiced your opinion on Slack.